samples/cellular/nrf_cloud_multi_service/src/cloud_connection.c

```diff
@@ -577,6 +594,9 @@ void cloud_connection_thread_fn(void)
 
 		LOG_INF("Network is ready");
 
+		/* Wait for a bit to give provisioning a chance */
+		k_sleep(K_SECONDS(15));
+
```

Comment:
Thanks to https://github.com/nrfconnect/sdk-nrf/pull/13186 this can (and should) be changed to be event-based.